Formula/qjackctl.rb

```diff
@@ -14,14 +14,23 @@ class Qjackctl < Formula
   depends_on "autoconf" => :build
   depends_on "qt5"
   depends_on "jack"
+  depends_on "portaudio" => :linked
+
+  # Fixes varaible length array error with LLVM/Clang
+  # Reported upstream: https://github.com/rncbc/qjackctl/issues/17
+  patch do
+    url "https://github.com/rncbc/qjackctl/pulls/18.patch"
+    sha256 "cbb7cc72e0086b8e6df24c8c3a462dc0e9297f095573adb7a3cf98502a1902d4"
+  end
 
   def install
     system "./autogen.sh"
     system "./configure", "--disable-debug",
                           "--enable-qt5",
                           "--disable-dbus",
                           "--disable-xunique",
-                          "--prefix=#{prefix}"
+                          "--prefix=#{prefix}",
+                          "--with-qt5=#{Formula["qt5"].prefix}"
```

Comment:
For me it's necessary (see #3278). I guess this is because `qt5` is not liked due to conflicts with `qt`...
I'll update the formula to use `opt_prefix` instead of `prefix`.